templates/zerver/api/update-display-settings.md

```diff
@@ -0,0 +1,35 @@
+# Update display settings
+
+{generate_api_description(/settings/display:patch)}
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|/settings/display:patch|example}
+
+{tab|curl}
+
+{generate_code_example(curl, include=["left_side_userlist", "emojiset"])|/settings/display:patch|example}
+
+{end_tabs}
+
+## Parameters
+
+{generate_api_arguments_table|zulip.yaml|/settings/display:patch}
+
+## Response
+
+#### Return values
+
+The server will return the settings that have been changed after the request,
+with their new value. Please note that this doesn't necessarily mean that it
+will return all the settings passed as parameters in the request, but only
+those ones that were different than the already existing setting.
```

Comment:
Structurally, this sort of paragraph isn't how we want our docs to work but see my comment about planned changes to this endpoint's structure that would rewrite how the return values work; given that plan, I think it's probably better to merge this roughly as-is. (I'll do a bit of copyediting).

Reply:
Do we want that format(mentioned on CZO) to be documented here now or,
Should I wait untill the plan you mentioned in [https://chat.zulip.org/#narrow/stream/3-backend/topic/update-subscription-properties/near/901757](https://chat.zulip.org/#narrow/stream/3-backend/topic/update-subscription-properties/near/901757) gets implemented?